src/Document/FilterElement.ts

```diff
@@ -15,6 +15,11 @@ export default class FilterElement extends Element {
 			document
 		} = this;
 		const boundingBox = element.getBoundingBox(ctx);
```

Comment:
when would this ever be missing?
`PathElement.getBoundingBox` is https://github.com/canvg/canvg/blob/master/src/Document/PathElement.ts#L110
which calls `path()` which always returns https://github.com/canvg/canvg/blob/master/src/Document/PathElement.ts#L106

what am i missing here?

Reply:
![image](https://user-images.githubusercontent.com/241819/70112682-f8ed3280-160a-11ea-9aeb-e200bac56218.png)

even typescript says it's always there

Reply:
@gabelerner 
1) `PathElement` is base class for some other elements, where `getBoundingBox` can be redefined
2) `getBoundingBox` can return `null`, not `undefined`.

Reply:
ah ok, i wish we can enforce it w/ types but kind of hard w/ polymorphism ... we can try to do some generic classes at some point like have a `PathElementWithBoundingBox` and a `PathElementWithoutBoundingBox` where they extend `PathElement<T>` where `T` is either a `BoundingBox` or `null` and then others can extend from there. for now will review rest of PR after i push out 3.0

Reply:
@gabelerner This fix should be in v3, there is text rendering fixes.

Reply:
ok i'll push out 3.0.1 w/ it no problem ... let me finish reviewing